universal-application-tool-0.0.1/app/services/question/types/NameQuestionDefinition.java

```diff
@@ -97,23 +97,23 @@ public QuestionType getQuestionType() {
   }
 
   public Path getFirstNamePath() {
-    return getPath().join("first");
+    return getPath().join("first_name");
```

Comment:
question: Why not use `Scalars.FIRST_NAME` here? Is that because these scalar path getters are all going to eventually be removed from `XQuestionDefinition` classes?

Reply:
these will get deleted soon.